codegenerator/model/api.go

```diff
@@ -74,20 +75,12 @@ import org.mozilla.taskcluster.client.TaskClusterRequestHandler;
 
     protected static final String defaultBaseURL = "` + api.BaseURL + `";
 
-    public ` + className + `(String clientId, String accessToken) {
-        super(clientId, accessToken, defaultBaseURL);
```

Comment:
We could, I guess. At the moment nothing is using the java library, so it didn't seem necessary to keep it for backwards compatibility, but I guess it is a simpler constructor if a user doesn't want to construct the Credentials object. I can add it back in, if you'd like it. I guess I liked the simplicity of having a single constructor, and only having "one way" to create a client.